lib/kibana/kibana/connector.py

```diff
@@ -194,6 +194,13 @@ def add_cookie(self, cookie):
         self.status = self.get('/api/status')
         self.authenticated = True
 
+    def add_api_key(self, api_key: str) -> bool:
```

Comment:
With this change, we need to bump the kibana version.

Reply:
Kibana version was not properly bumped - https://github.com/elastic/detection-rules/blob/0c25cfb82e556063fe3d779f5e73fddffea55975/lib/kibana/kibana/__init__.py#L11

cc: @eric-forte-elastic